src/Omnipay/Common/Message/AbstractResponse.php

```diff
@@ -148,6 +148,16 @@ public function getTransactionReference()
     }
 
     /**
+     * Get the transaction ID as generated by the merchant website.
+     *
+     * @return string
+     */
+    public function getTransactionId()
```

Comment:
My guess is that this should be an abstract function without a body or be included in the interface. I take it null is never the response people actually want, right?